src/ibek/ioc.py

```
@@ -45,6 +45,9 @@ class Entity(BaseSettings):
         description="enable or disable this entity instance", default=True
     )
     _model: EntityModel
+    # list of sub-entities
+    _child_entities: list[Entity] = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)